bot/exts/moderation/infraction/_utils.py

```diff
@@ -31,8 +30,10 @@
 
 APPEAL_EMAIL = "appeals@pythondiscord.com"
 
-INFRACTION_TITLE = f"Please review our rules over at {RULES_URL}"
-INFRACTION_APPEAL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_TITLE = "Please review our rules"
+INFRACTION_APPEAL_EMAIL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_APPEAL_MODMAIL_FOOTER = ('If you would like to discuss or appeal this infraction, '
+                                    'send a message to the ModMail bot')
```

Comment:
We use this style of multiline strings throughout the code base.
```suggestion
INFRACTION_APPEAL_MODMAIL_FOOTER = (
    'If you would like to discuss or appeal this infraction, '
    'send a message to the ModMail bot'
)
```

Reply:
That's fine. Perhaps we could add that rule to the pre-commit checks?